src/commands/email/send.js

```diff
@@ -153,7 +216,7 @@ class Send extends BaseCommand {
 
   async promptForText() {
     this.emailText = this.flags.text;
-    if (!this.emailText) {
+    if (this.isTTY && !this.emailText) {
```

Comment:
This is basically duplicated code. Any thoughts about moving this into a helper function passing in what to validate and what to prompt for?